packages/model-registry/contract-tests/jest.setup.js

```diff
@@ -0,0 +1,5 @@
+// Jest setup file for contract tests
+// This ensures Jest types are loaded properly for contract testing
+
+// Import the contract test setup to initialize Jest matchers
+require('@odh-dashboard/contract-tests');
```

Comment:
Remove the jest.config.js and jest.setup.js if they aren't used for your use case. We can followup to fix running vanilla jest later.